inbox/introduction.mdx

```diff
@@ -1,44 +1,33 @@
 ---
-title: "Introduction"
+title: 'Introduction'
 description: "Learn more about Novu's Inbox Component."
 ---
 
 The Inbox component, also sometimes referred to as a widget that furnishes your web application with a beautiful and smooth Inbox notification experience for users.
 
 In-App notifications are different from push notifications in the sense that Inbox notifications are consumed by the user in your app itself, whereas push notifications are shown on the device level. Read more about [push channel and supported providers.](/channels-and-providers/push/overview)
 
-Novu provides a pre-built, ready-to-use, and customizable UI component. It also allows you to bring your own UI via a headless component which can be used to add In-App notifications to any platform.
+Novu provides a pre-built, ready-to-use, and customizable Inbox UI component.
 
 ## Example
 
-[![Edit kind-nova-xv6s9g](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/kind-nova-xv6s9g?fontsize=14&hidenavigation=1&theme=dark)
+[![Edit kind-nova-xv6s9g](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/devbox/pensive-ritchie-cjwvc5)
```

Comment:
I created the new codesandbox with new Inbox React component

Reply:
Wdyt about using Stackblitz instead? It boots much faster